File: L004/benjaminBulbs.java
import java.util.Scanner;

public class benjaminBulbs {
    public static Scanner sc = new Scanner(System.in);

    public static void benjaminBulbs_(int n) {
        for (int i = 1; i * i < n; i++) {
            System.out.println(i * i);
        }
    }

    public static void main(String[] args) {

        int noOfFluctuations = sc.nextInt();

        benjaminBulbs_(noOfFluctuations);
    }
}

File: L004/benjaminBulbs.cpp
#include<iostream>
using namespace std;


void benjaminBulbs(int n){
    for(int i=1;i * i < n ;i++){
        cout<<i*i<<endl;
    }
}


int main(){

   int noOfFluctuations;
   cin>>noOfFluctuations; 

   benjaminBulbs(noOfFluctuations);
    
    return 0;
}